pkg/reconciler/revision/cruds.go

```diff
@@ -46,6 +48,20 @@ func (c *Reconciler) createDeployment(ctx context.Context, rev *v1.Revision) (*a
 	return c.kubeclient.AppsV1().Deployments(deployment.Namespace).Create(ctx, deployment, metav1.CreateOptions{})
 }
 
+func (c *Reconciler) createSecret(ctx context.Context, ns *corev1.Namespace) (*corev1.Secret, error) {
+	secret := &corev1.Secret{
```

Comment:
I guess data are filled in by the control-plane reconciler.

Reply:
There is a doc, but I'm not sure it specifies this level of detail.

I'm slightly confused about where the secret comes from here, and what the lifecycle is if internal encryption is disabled. (It may be that it's fine to drop this resource, but I don't quite understand the lifecycle.)

Reply:
The lifecycle of the secret in user's ns (queue-proxy's server cert) is:
1. The secret is created when users deploy the first Kservice in the namespace.
2. The secret is shared by all Kservice in the namespace.
3. The secret is dropped when the namespace is deleted. (the secret is neither dropped when all Kservice is deleted in the namespace nor the internal encryption is disabled.).

There are many ways to manage the secret in user's ns and so I chose the simplest way for now.